src/unity/toolkits/coreml_export/neural_net_models_exporter.cpp

```diff
@@ -291,31 +291,29 @@ std::shared_ptr<MLModelWrapper> export_activity_classifier_model(
   ModelDescription* model_desc = model.mutable_description();
 
   // Write the primary input features.
-  set_array_feature(model_desc->add_input(), "features",
-                    get_activity_classifier_features_short_description(
-                        prediction_window, features),
-                    { 1, prediction_window, features.size() });
+  for (size_t i = 0; i < features.size(); i++) {
+    set_array_feature(model_desc->add_input(), features[i],
+                      get_activity_classifier_features_short_description(
+                          prediction_window, features[i]),
+                      {1, 1, prediction_window});
```

Comment:
Wait, could we make the shape just `{prediction_window}` here instead? Does that still work without adding in a reshape layer?

Reply:
I will try it and tell you.